cal/decoder/library.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "cal-decoder",
+    "version": "1.0.0",
+    "description": "Decodes CAN bus message data from the M150 and PDM15",
+    "keywords": "CAN",
+    "authors":
+    [
+      {
+        "name": "Ethan Peterson",
+        "email": "ethan@petetech.net",
+        "url": "portfolio.petetech.net"
+      },
+      {
+        "name": "Andrew Smith",
```

Comment:
@Jchisholm204 @mcassar4 Can you guys update these author fields? Please have everyone in the CAN group add their name! Great work on all of this!